pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -32,6 +32,8 @@
 , lib32 ? null
   # Whether to extract the GSP firmware
 , firmware ? openSha256 != null
+  # Override whether this includes IBT-compatible binary blobs
+, hasIbtSupport ? null
```

Comment:
Or just name it `ibtSupport` and default to false?

Reply:
I was trying to have it indicate that `null` = default behavior, but now realizing that doesn't really work. Will update in a minute